app/views/projects/_form.html.erb

```diff
@@ -8,6 +8,12 @@
     <%= form.input :repository_url, required: true %>
     <%= form.input :permalink, required: true if project.persisted? %>
     <%= form.input :release_branch, help: 'New commits on this branch will cause a release when a webhook arrives.' %>
+    <div class="form-group">
```

Comment:
best to add some help text like 'select which type of webhook will trigger a release'